src/measure_theory/measure_space.lean

```diff
@@ -246,6 +247,20 @@ ext $ λ s, begin
       (m₂.mono' (inter_subset_right _ _))⟩,
 end
 
+theorem trim_smul (c : ennreal) {m : outer_measure α}
+  (hm : ∀ s, m.trim s = 0 → ∃ t, s ⊆ t ∧ is_measurable t ∧ m t = 0) :
+  (c • m).trim = c • m.trim :=
```

Comment:
In #3195 I have an unconditional proof of this (without `hm`), which is quite ugly though.

The condition is also true for all outer measures, see `exists_is_measurable_superset_of_trim_eq_zero` in the same PR.

So presumably we want my proof of `exists_is_measurable_superset_of_trim_eq_zero` and your proof of `trim_smul`.

Reply:
I can replace the proof while merging conflicts.